### ISSUE TITLE
BUG - [High] - [iOS] - Blind Veteran cannot select an option from the clinician drop down list in secure messaging, other messaging-related issues

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va-mobile-app#10414]
## Specs:

- Device: iPhone 15 Pro
- OS:
- User Account:
- Accessibility State:

## Steps to Reproduce

This comes from one of the members of Blinded Veterans Association who contacted me directly: 

"When I secure message a doctor I’m able to select the clinician and hit the down button but when you get to the subject area where you have to select one, the Cancel and Done button or one so it doesn’t let you select so I have not been able to send my doctors messages and this has been at least two weeks. And that’s a problem because I’m currently under going treatment and so I really need to get a hold of my doctors."

## Desired behavior

This VoiceOver user should be able to select the clinician from the list successfully and so they can compose and send their secure message.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High 
- Frequency: Unknown 

- 2 - High

Marking this High priority since patient safety is impacted. I would have chosen Critical for this reason but I'm not sure if all iPhone users are experiencing this bug. 

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [x] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by artsymartha68** [comment]
Updates from the Veteran:

1. Reinstalling the app appeared to fix the original issue with the drop down box.
2. "However, when I open an  older message, it won’t take you back at the top left corner where it says review message there’s no link. You actually have to click down at the bottom tab where it says health for it to go back so I don’t know if you were aware of that is happening in my particular situation once I’m in the message for the secure message."
3. "Also, in the health portion where it shows messages I have this number four next to the word, but all of my messages have been open and read, it does not go away. It will add another number if there is an red one, so it will be five or six etc., those four do not go away Even though I went through everything and there are no one open messages."

**Comment by SKMOstudio** [comment]
I have tested this issue and found a few choke points that might make the experience difficult for screen reader users:

https://github.com/user-attachments/assets/6d9a19c0-cefa-48a9-a1b4-60a61e5522c3

**Comment by mmutia** [comment]
Hi @SKMOstudio! Is it possible that I can use your test creds? Additionally, where exactly in the app are these issues occurring? I am a native screen reader user as well and would love to take a look!

**Comment by SKMOstudio** [comment]
Hi @mmutia , I used the DEMO user login to test this. From the Home screen > Select the _Messages_ link > On the Messages page, select the _Start new message_ button.

**Comment by mmutia** [comment]
Thank you @SKMOstudio!

I agree with what you noted above -- the difficulties that the Veteran is experiencing is largely a consequence of the usability issues / bugs that are occurring. However, I am having trouble replicating this specific issue that the Veteran mentioned:

"Also, in the health portion where it shows messages I have this number four next to the word, but all of my messages have been open and read, it does not go away. It will add another number if there is an red one, so it will be five or six etc., those four do not go away Even though I went through everything and there are no one open messages."

**Comment by SKMOstudio** [comment]
Closing this issue.

I am not able to replicate the issue described by the user, as I can open the option menu, select an available option, and then close or confirm the selection. However, I did find 2 additional issues I have written up in separate bug tickets:

#10931  - Exiting an option menu causes screen reader for focus on 'Back' button at top of page
#10933 - User in multi-line text field has difficulty exiting keyboard
